### DISCUSSION TITLE
Can ASReview LAB be used for things other than systematic reviews?

### DISCUSSION THREAD

**Discussion by SagevdBrand** [asreview/asreview#560] [Q&A] [answered]
All of the documentation is about using ASReview for systematic reviews, but are there any other use-cases in which the software can be used? Thinking about emails, or archive files? Anybody any ideas?

**Comment by LarsTummers** [comment, marked as answer]
Yes it can. I have used it to systematically search for relevant newspaper articles. You can just load in the articles using title and abstract. Title = newspaper article title. Abstract = newspaper article body text.
